bsip-0074.md

```diff
@@ -24,13 +24,89 @@ To ensure the debt positions can always be closed with suffcient smartcoin, it i
+
+<br>
+
+`limit_order_receives = X / match_price`
+
+`call_order_relinquishes = (X / match_price) * [MSSR / (MSSR - MCFR)]`
+
+`margin_call_fee = call_order_relinquishes - limit_order_receives`.
+
+<br>
+
+This calculation results in a Margin Call Fee which is proportional to the amount of collateral paid to the limit order to buy _X_ amount of smartcoin.
+
+#### Calculation of fee, maker case:
+
+Note that in the case when the margin call enters the book as a maker order (i.e., when it was not able to immediately find an existing favorable limit order), the match price will be identical to the order price, which allows the forgoing calculations to simplify as follows:
+
+<br>
+
+`limit_order_receives = X * (MSSR - MCFR) / median_feed_price`
+
+`call_order_relinquishes = X * MSSR / median_feed_price = X / max_short_squeeze_price`
+
+`margin_call_fee = X * MCFR / median_feed_price`
+
+<br>
+
+The above simplification offers two insights into the Margin Call Fee:
+
+1. The reason why Maximum Short Squeeze Price is retained as the Global Settlement threshold is because this is the price that determines the total amount of collateral a call order must relinquish under least favorable conditions, and,
+
+2. That the MCFR can be interpreted as defining a fraction of the feed price that establishes the maximum fee rate paid by the callee.  (I.e., the fee will not exceed `MCFR * X / median_feed_price`.)
+
+
+### Fee Floor
+
+Because the effect of the Margin Call Fee is to reduce the amount of collateral offered by the margin call to buy back debt, there is a danger that the margin call could offer less collateral than the "even exchange" value of the debt (as reported by the price feed publishers), making it unlikely to match the call with a buyer.  To prevent this, the implementation should include a floor wherein every place that `MSSR - MCFR` appears, it is replaced with:
+
+`(MSSR - MCFR)` —> `max((MSSR - MCFR), 1.000)`.
+
+This will ensure that the amount of collateral offered by the margin call is never valued less than the quantity of debt which the call seeks to buy.  At the same time, however, it may mean that the asset issuer receives less collateral (or even no collateral) than they are expecting in fees.  Asset issuers should be advised not to expect to collect a fee rate greater than (MSSR - 1.000).
+
+### Specification summary
```

Comment:
I think the description below is sufficient for a non-technical summary, so best to just rename this section to `# Non-technical Summary` and remove the "Shareholder Summary" section below.